action.yml

```diff
@@ -32,6 +32,9 @@ inputs:
   annotate:
     description: 'Whether or not to attempt to create GitHub annotations to show test failures inline. Only effective on Julia 1.8+.'
     default: 'false'
+  compiled_modules:
+    description: 'Whether to run tests with `compiled-modules` equal to `yes` or `no`'
+    default: 'yes'
```

Comment:
Julia can also take "existing" as an option

Reply:
Weirdly this is not documented in julia --help 🤔 Is that an oversight.

Reply:
It's on 1.11-DEV https://docs.julialang.org/en/v1.11-dev/manual/command-line-interface/#command-line-interface

Reply:
Perhaps just don't specify the accepted values and point users to the julia docs?

Reply:
Ah, that makes sense. Okay!